### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by renitae**
# Why is this needed?
Adds on to the story

Makes it more interesting with more details

### CODE REVIEW

**Review by GabiCar2 — APPROVED** [review]
i like the specified cereal so i approve of this

**Comment by AlisaSalikhov** [comment]
OH NO NOT THE CINNAMON CRUNCH 😭

File: README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ Also, one of them had a very, very BIG secret; she could turn into a cupcake if
 Anyways, back to the story…
 They were both going shopping one day then one of them got lost.
 They also forgot where they parked their shopping cart with all of their merchandise in it.
+This was a major problem, as they had their cinnamon toast crunch cereal in their shopping cart and needed to find it before breakfast.
```
Comment (xlgmokha) [inline README.md:10]: I love Cinnamon Toast Crunch!

**Review by delilah133 — APPROVED** [review]
That is a major problem, I agree.

**Pull request merged**